docs/docs/index.md

```diff
@@ -1,36 +1,24 @@
-# Welcome
+# **Welcome**
 
-Hydra is a layer-two scalability solution for Cardano, which aims to increase
-the speed of transactions (low latency, high throughput) and minimize
-transaction cost.
+This section is a user manual for Hydra node operators. 
 
-This is the user manual for `hydra-node`, an implementation of the Hydra Head protocol, as originally published [here](https://eprint.iacr.org/2020/299.pdf). The `hydra-node` interfaces the Cardano blockchain, connects to other `hydra-node`s on a dedicated overlay network, runs a simplified (coordinated) Hydra Head protocol, and provides an API to clients.
+Hydra node is an implementation of the Hydra Head protocol, designed to enable fast and low-cost transactions by operating off-chain while leveraging the security of the main Cardano blockchain.
 
-:::warning Mainnet Availability disclaimer
+Decentralized applications (DApps), exchanges, and enterprise-level services can choose to run a Hydra node to participate in forming Hydra Heads – mini-ledgers that process transactions independently and in parallel, significantly enhancing throughput and reducing latency. 
 
-The Hydra Head protocol version 0.10.0 or newer is compatible with the Cardano
-mainnet, which means it is possible to run a `hydra-node` on mainnet using real
-funds.
+To run a Hydra node, participants should:
 
-Before running a `hydra-node` to take part in the Hydra Head protocol,
-developers are strongly encouraged to review the [known issues][known-issues] in
-the documentation in order to understand the current limitations and the
-possible consequences.
+* **Install and configure the node**: set up the Hydra node software, ensuring it is properly configured to connect with other Hydra nodes and the Cardano mainnet.
+* **Participate in Hydra heads**: collaborate with other nodes to create and manage Hydra heads, enabling batch processing of transactions off-chain.
+* **Synchronize with Cardano**: maintain synchronization with the main chain to ensure the security and finality of transactions processed by the Hydra heads.
 
-By using Hydra Head protocol version 0.10.0 or newer, you understand the
-protocol is in development and that use of the `hydra-node` on mainnet is
-entirely at your own risk.
+Navigate through tutorials and documentation guides to get started. If you want to learn more about Hydra, see this section. If you're interested in building, see this section.
```

Comment:
Please add links to the 'core concepts' section and 'development docs'

Reply:
Updated